aeat.py

```diff
@@ -336,58 +339,26 @@ def is_decimal(value):
             cursor.execute(query)
             result = cursor.fetchall()
 
-            tax_identifiers = [r[0] for r in result]
-            parties = dict((p, {}) for p in tax_identifiers)
-            for tax_identifier in PartyIdentifier.search(
-                        [('code', 'in', tax_identifiers)]):
-                party = Party.search([
```

Comment:
As you no longer use "Party" you should remove the instantiation of the class at the top of the function.

Again, pyflakes detects these cases.

Reply:
Done.